var/spack/repos/builtin/packages/py-dnaio/package.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyDnaio(PythonPackage):
+    """Read and write FASTQ and FASTA"""
+
+    homepage = "https://github.com/marcelm/dnaio"
+    url      = "https://pypi.io/packages/source/d/dnaio/dnaio-0.3.tar.gz"
+    git      = "git@github.com:marcelm/dnaio.git"
```

Comment:
```suggestion
    git      = "https://github.com/marcelm/dnaio.git"
```

Reply:
provide git url with https, not ssh, protocol